src/cpp/runner/Benchmark.h

```diff
@@ -37,35 +37,48 @@ void call_member_function(std::unique_ptr<ITest<Input, Output>>& ptr_to_object,
 template<class Input, class Output>
 unique_ptr<ITest<Input, Output>> get_test(const ModuleLoader& module_loader) = delete;
 
-//Measures time according to the documentation.
 template<class Input, class Output>
-double measure_shortest_time(const double minimum_measurable_time, const int nruns, const double time_limit, std::unique_ptr<ITest<Input, Output>>& test, const test_member_function<Input, Output> func)
+int find_repeats_for_minimum_measurable_time(const duration<double> minimum_measurable_time,
+    std::unique_ptr<ITest<Input, Output>>& test,
+    const test_member_function<Input, Output> func, duration<double>& min_sample,
```

Comment:
Are these reference args output args?  If so, please pass by pointer, else by value or const ref.

Reply:
I've replaced it by returning a struct.